yt/visualization/tests/test_plotwindow.py

```diff
@@ -406,6 +406,73 @@ def test_creation_with_width(self):
             assert_true(aun == plot._axes_unit_names)
 
 
+class TestPerFieldConfig(unittest.TestCase):
+
+    ds = None
+
+    def setUp(self):
+        from yt.config import ytcfg
+
+        newConfig = {
+            ("yt", "default_colormap"): "viridis",
+            ("plot", "gas", "log"): False,
+            ("plot", "gas", "density", "units"): "lb/yard**3",
+            ("plot", "gas", "density", "path_length_units"): "mile",
+            ("plot", "gas", "density", "cmap"): "plasma",
+            ("plot", "gas", "temperature", "log"): True,
+            ("plot", "gas", "temperature", "linthresh"): 100,
+            ("plot", "gas", "temperature", "cmap"): "hot",
+            ("plot", "gas", "pressure", "log"): True,
```

Comment:
Do we want to also add a test here where `linthresh` is set but log isn't?

Reply:
Yes we do, since doing it uncovered a bug related to your comment (https://github.com/yt-project/yt/pull/1931#discussion_r602553624)!

Reply:
!!!! Ah I'm glad my last review helped!